lib/Crust/Request.pm6

```diff
@@ -84,7 +84,7 @@ method header(Str $name) {
 method content() {
     # TODO: we should support buffering in Crust layer
     my $input = $!env<p6sgi.input>;
-    $input.seek(0,0); # rewind
+    $input.seek(0); # rewind
```

Comment:
Ah, the `IO::Handle` is used in the test, because usually `IO::Blob` is used, it is no good just this fix.
Interface of `IO::Handle` and `IO::Blob` of seek method is because the mismatch at the moment, is better to correct the `IO::Blob`?